kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -1168,6 +1183,24 @@ private void convertRule(RuleOrClaim rule, int ruleIndex, boolean heatCoolEq, St
             sb.append("\n  ");
             convert(consideredAttributes, rule.att(), sb, freeVarsMap, rule);
             sb.append("\n\n");
+        } else {
+            assertNoExistentials(rule, existentials);
+            sb.append("  axiom{R");
+            Option<Set> sortParams = rule.att().getOption("sortParams", Set.class);
+            if (sortParams.nonEmpty()) {
+                for (Object sortParamName : sortParams.get())
+                    sb.append("," + sortParamName);
+            }
+            sb.append("} ");
+            sb.append("\\equals{");
+            sb.append(ruleInfo.productionSortStr);
+            sb.append(",R} (\n    ");
+            convert(left, sb);
+            sb.append(",\n    ");
+            convert(right, sb);
+            sb.append(")\n  ");
+            convert(consideredAttributes, rule.att(), sb, freeVarsMap, rule);
+            sb.append("\n\n");
```

Comment:
It looks like here we're including the macro rules in the final definition.kore (in addition to putting them in macros.kore). Is that correct? What's the reasoning?

Reply:
No, if you look at the above if condition, we are controlling which file we emit rules to based on whether that rule is a macro.

Reply:
Ah sorry, I missed that this code was inside `convertRule`. Thanks!